packages/amplication-data-service-generator/src/server/static/src/auth/gqlUserRoles.decorator.ts

```diff
@@ -1,16 +0,0 @@
-import { createParamDecorator, ExecutionContext } from "@nestjs/common";
-import { GqlExecutionContext } from "@nestjs/graphql";
```

Comment:
Please explain why do you think we do not need this file and what is the alternative